### ISSUE TITLE
Create add Note and add Password UI Screens

### ISSUE THREAD

**Issue by vishnuagbly** [vishnuagbly/pass-man#7]
We need to have screens of adding passwords and adding note screens, for this might also need to add floating action button and whatever animation-like the thing to get two small buttons from a bigger button, basically according to the wireframe.

From Wireframe should get the idea about how will transmission from the home screen to the add a password screen and add a note screen will take place.

Will also need to add different files for both inside the screens folder.

On both screens just left the onChanged and onSubmit properties of textfield, or just add someplace where we will call functions to make API calls to check password strength, another to get auto websites logo and preferably will show auto-complete when typing URL. And then finally we will call a function on submit button.

Basically, just create the UI for now, we will assign variables and function calls later.
